Functions/Program.cs

```diff
@@ -0,0 +1,7 @@
+using Microsoft.Extensions.Hosting;
+
+var host = new HostBuilder()
+    .ConfigureFunctionsWorkerDefaults()
+    .Build();
+
+host.Run();
```

Comment:
We should run this asynchronously.
i.e:
`await host.RunAsync();`

Reply:
Done